css/style.css

```diff
@@ -0,0 +1,21 @@
+nav {
+    display: flex;
+    justify-content: flex-end;
+    position: fixed;
+    top: 0;
+    right: 0;
+    width: 100%;
+    background: var(--main-red);
+  }
+  
+  #header-img {
+    width: 20%;
+    height: 10%;
+    box-sizing: border-box;
+  }
+  
+  @media (max-width: 768px) {
+   html {
+     font-size: 78.5%;
+   } 
+  }
```

Comment:
Good job using the right spacing and newlines for your stylesheet codes. However, as a result of not having linters in your work you were adapting the 4 spacing indentation method which is currently not the standard, I advise you install the necessary linters to your code editors.